db/schema.rb

```diff
@@ -12,7 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2020_03_20_105826) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
+
+  create_table "form_responses", force: :cascade do |t|
+    t.json "form_response", default: {}, null: false
```

Comment:
Can we get this to use `jsonb` instead of `json`?

Reply:
Maybe it already does, I don't know rails lol

Reply:
Sure 👍

Reply:
Done.  It won't use jsonb unless we explicitly state it.